overlord/storecontext/context.go

```diff
@@ -69,9 +69,13 @@ type DeviceSessionRequestSigner interface {
 	SignDeviceSessionRequest(serial *asserts.Serial, nonce string) (*asserts.DeviceSessionRequest, error)
 }
 
-type ProxyStoreer interface {
+type StoreOptions interface {
 	// ProxyStore returns the store assertion for the proxy store if one is set.
 	ProxyStore() (*asserts.Store, error)
+
+	// StoreAccess returns a string indicating whether the store should have
+	// network access or not
+	StoreAccess() (string, error)
```

Comment:
Based on this interface alone, it's not clear what values this can return. If this returned a specific type based on an iota instead of a string, the values that can be returned would be constrained which would be clearer and less error prone

Reply:
Right now, we really treat it as a boolean. `offline`, or unset. Do you think a boolean would be better?

Reply:
I'm fine with either introducing an enumeration or using StoreOffline() bool for the time being

Reply:
Done, thanks!